### PR TITLE
added two code examples: add_local_campaign.rb and handle_rate_exceeded_error.rb

### PR DESCRIPTION
**Pull request by xflesym**
examples/advanced_operations/add_local_campaign.rb
	examples/error_handling/handle_rate_exceeded_error.rb

### CODE REVIEW

File: examples/advanced_operations/add_local_campaign.rb [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  end
+
+  # Issues a mutate request to add campaign budget.
+  response = client.service.campaign_budget.mutate_campaign_budgets(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  budget_resource_name = response.results.first.resource_name
+  puts "Created campaign budget with resource name: #{budget_resource_name}"
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:67]: Add single quote around a string (budget resource name)

File: examples/advanced_operations/add_local_campaign.rb [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  end
+
+  # Issues a mutate request to add campaign.
+  response = client.service.campaign.mutate_campaigns(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  campaign_resource_name = response.results.first.resource_name
+  puts "Created Local campaign with resource name: #{campaign_resource_name}"
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:116]: Ditto

File: examples/advanced_operations/add_local_campaign.rb [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  end
+
+  # Issues a mutate request to add the ad group.
+  response = client.service.ad_group.mutate_ad_groups(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  ad_group_resource_name = response.results.first.resource_name
+  puts "Created ad group with resource name: #{ad_group_resource_name}"
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:140]: Ditto

File: examples/advanced_operations/add_local_campaign.rb [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+  end
+
+  # Issues a mutate request to add the ad group ad.
+  response = client.service.ad_group_ad.mutate_ad_group_ads(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  puts "Created ad group ad with resource name: " \
+    "#{response.results.first.resource_name}"
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:193]: Ditto

File: examples/advanced_operations/add_local_campaign.rb [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+  # Issues a mutate request to add the asset.
+  response = client.service.asset.mutate_assets(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  # Prints out information about the newly added asset.
+  asset_resource_name = response.results.first.resource_name
+  puts "A new image asset has been added with resource name: " \
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:214]: Ditto

File: examples/advanced_operations/add_local_campaign.rb [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+
+  # Issues a mutate request to add the asset.
+  response = client.service.asset.mutate_assets(
+    customer_id: customer_id,
+    operations: [operation],
+  )
+
+  # Prints out information about the newly added asset.
+  asset_resource_name = response.results.first.resource_name
+  puts "A new YouTube video asset has been added with resource name: " \
```
Comment (fiboknacky) [inline examples/advanced_operations/add_local_campaign.rb:243]: Ditto

**Review by fiboknacky — CHANGES_REQUESTED** [review]

File: examples/advanced_operations/add_local_campaign.rb (hunk 1 above)
Comment (xflesym) [reply examples/advanced_operations/add_local_campaign.rb]: Done.

**Review by fiboknacky — APPROVED** [review]

File: examples/error_handling/handle_rate_exceeded_error.rb [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+require 'google/ads/google_ads'
+require 'date'
+
+def handle_rate_exceeded_error(customer_id, ad_group_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # Sequentially sends the requests.
+  for i in 0..NUM_REQUESTS-1
```
Comment (mcloonan) [inline examples/error_handling/handle_rate_exceeded_error.rb:34]: NUMREQUESTS.times do |i|

File: examples/error_handling/handle_rate_exceeded_error.rb [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+end
+
+def create_ad_group_criterion_operations(
+  client,
+  customer_id,
+  ad_group_id,
+  req_index
+)
+  operations = []
+  for i in 0..NUM_KEYWORDS-1
```
Comment (mcloonan) [inline examples/error_handling/handle_rate_exceeded_error.rb:90]: Same here.

**Review by mcloonan — CHANGES_REQUESTED** [review]

File: examples/error_handling/handle_rate_exceeded_error.rb (hunk 7 above)
Comment (xflesym) [reply examples/error_handling/handle_rate_exceeded_error.rb]: Done.

File: examples/error_handling/handle_rate_exceeded_error.rb (hunk 8 above)
Comment (xflesym) [reply examples/error_handling/handle_rate_exceeded_error.rb]: Done.

**Review by mcloonan — APPROVED** [review]

**Pull request merged**